examples/app-menu/content.ts

```diff
@@ -128,7 +129,10 @@ InboxSDK.load(2, 'app-menu').then(async (sdk) => {
 
   panel1.addNavItem({
     name: 'Nav Item 1',
-    onClick: () => sdk.Router.goto('custom-route-3'),
+    routeID: ['custom-route-1', 'custom-route-4'],
```

Comment:
This API is a bit awkward because
- not all of the values are used for the same purposes. All of them are used for checking if the nav item is active, but only the first one is used as the default link.
- You can pass multiple route IDs but not multiple route params (and it looks like the implementation entirely ignores route params during nav item active checking if multiple route IDs are passed).

I think a better API would be to add a `secondaryRoutes?: Array<{routeID: string; routeParams?: Record<string, string|number>}>` property to `NavItemDescriptor` which are only used for active checking, and then make the implementation of `_handleRouteViewChange` in nav-item-view.ts be:

```ts
function _handleRouteViewChange(
  navItemViewDriver: GmailNavItemView,
  [navItemDescriptor, routeViewDriver]: [NavItemDescriptor, RouteViewDriver]
) {
  let isActive = false;
  if (navItemDescriptor) {
    const routesToCheck = [
      {
        routeID: navItemDescriptor.routeID,
        routeParams: navItemDescriptor.routeParams,
      },
      ...(navItemDescriptor.secondaryRoutes || []),
    ];
    isActive = routesToCheck.some(
      (route) =>
        routeViewDriver.getRouteID() === route.routeID &&
        isEqual(routeViewDriver.getParams(), route.routeParams || {})
    );
  }
  navItemViewDriver.setActive(isActive);
}
```